src/main/webapp/i18n/en/exam.json

```diff
@@ -241,6 +241,8 @@
             }
         },
         "examManagement": {
+            "reviewDatesInvalidWarning": "End of Student Review is set incorrect or missing",
+            "reviewDatesInvalidExplanation": "If the Start of Student Review is set, the End of Student Review has to be set as well. Furthermore, the End of Student Review has to be chronologically after its start.",
```

Comment:
I would consider using the term "Begin" as the field has the label "Begin of Student Review". And if you prefer capitalization of start and end I would capitalize every occurrence, so here the last word (start / begin) could be capitalized as well.

Reply:
Thanks for the hint, I made it consistent in wording and capitalization.